src/Microsoft.Azure.NotificationHubs.DotNetCore.Tests/ManagementApiE2ETests.cs

```diff
@@ -99,6 +123,44 @@ public void ExecuteCreateGetAndDeleteNotificationHubMethods_ShouldCreateGetOrDel
             CleanUp();
         }
 
+        [Fact]
+        public void SubmitAndGetNotificationHubJob_ShouldReceiveCorrectJobs()
+        {
+            CleanUp();
+
+            _namespaceManager.CreateNotificationHub(_notificationHubName);
+
+            var blobClient = new CloudBlobClient(
+                _storageEndpoint,
+                new StorageCredentials(_storageAccount, _storagePassword));
+
+            var container = blobClient.GetContainerReference(_containerName);
+
+            var outputContainerSasUri = GetOutputDirectoryUrl(container);
+            var inputFileSasUri = GetInputFileUrl(container, InputFileName);
+
+            var notificationHubJob = new NotificationHubJob()
```

Comment:
Nit that `()` are not required with empty constructors when doing object initializers